.github/workflows/common-terraform-module-ci-v2.yml

```diff
@@ -59,10 +59,17 @@ jobs:
     uses: ./.github/workflows/post-usage.yml
     secrets: inherit
 
+  test_message:
```

Comment:
remove this test

Reply:
Sure, thank you for pointing out. 
As part of trying to verify I might have missed this.Will remove this.